helix-view-aggregator/src/test/java/org/apache/helix/view/integration/TestHelixViewAggregator.java

```diff
@@ -125,7 +123,8 @@ public void testHelixViewAggregator() throws Exception {
 
     // Wait for refresh and verify
     Thread.sleep((_viewClusterRefreshPeriodSec + 2) * 1000);
-    verifyViewClusterEventChanges(false, true, true);
+    verifyInstanceConfigChange();
```

Comment:
This as well as all the other similar changes:

The behavior is changed here. When it was (false, true, true), it was verifying that `_monitor.getExternalViewChangeCount() <= 0`, but that's no longer a check here. Is there a reason for that?

Reply:
You are right about the behavior change, we are narrowing down the test scope since "No cluster event change" is actually not a functional requirement. 
On the other side, the overlapping effect is also introducing noise to the overall CI tests.

Reply:
Yes. In extreme scenario, cluster event triggered by previous test can happen after the monitor reset and received during the following test, this would cause unexpected test failure if we check `_monitor.getExternalViewChangeCount() <= 0` e.g.
And yes, there could be false negative in some cases (regardless of this patch or not), where `> 0` test pass because of events from previous test or other sources.